site/docs/operations/docker.md

```diff
@@ -0,0 +1,340 @@
+
+```bash
+docker run hazelcast/hazelcast-jet
+```
+
+And submit the job, specifying the cluster address:
+
+```bash
+bin/jet -a 172.17.0.2 submit examples/hello-world.jar
+```
+
+## Configure Hazelcast Jet
+
+### JAVA_OPTS
+
+As shown below, you can use `JAVA_OPTS` environment variable if you need
+to pass multiple VM arguments to your Hazelcast Jet member.
+
+```bash
+docker run -e JAVA_OPTS="-Xms512M -Xmx1024M" -p 5701:5701 hazelcast/hazelcast-jet
+```
+
+### Custom Hazelcast Jet Configuration File
+
+If you need to configure Hazelcast Jet with your own hazelcast-jet.yaml,
+or hazelcast.yaml, you just need to mount the file to configuration file
+location in the container. The configuration files are stored in the
+`/opt/hazelcast-jet/config` directory.
+
+For example to change the cluster name create a file `hazelcast.yaml`
+with following content:
+
+```yaml
+hazelcast:
+  # The name of the cluster. All members of a single cluster must have the
+  # same cluster name configured and a client connecting to this cluster
+  # must use it as well.
+  cluster-name: jet-1
+```
+
+Then use it when starting the Jet instance by running the following
+command (it expects the file in current directory):
+
+```bash
+docker run -v "$(pwd)"/hazelcast.yaml:/opt/hazelcast-jet/config/hazelcast.yaml hazelcast/hazelcast-jet
+```
+
+Submit a job to the cluster, specifying cluster name in the jet command
+using `-n jet-1` option (shortcut for `--cluster-name`):
+
+```bash
+docker run -it -v "$(pwd)"/examples:/examples hazelcast/hazelcast-jet jet -a 172.17.0.2 -n jet-1 submit /examples/hello-world.jar
+```
+
+## Start Jet cluster
+
+While running a cluster locally will not give you any computational
+advantage, it will allow you to test your pipelines in a distributed
+environment.
+
+The simplest way to start a cluster is to start multiple docker
+instances. Just run the `docker run` command from multiple terminals,
+remember to use different port each time:
+
+```bash
+docker run -p 5701:5701 hazelcast/hazelcast-jet
+...
+docker run -p 5702:5701 hazelcast/hazelcast-jet
+```
+
+Eventually, the instances should discover each other and you should
+see output similar to:
+
+```text
+Members {size:2, ver:2} [
+    Member [172.17.0.2]:5701 - df428435-8cbd-464d-8671-afe396f5eef6
+    Member [172.17.0.3]:5701 - 011a9dd4-936f-4170-a7c6-622b7430b789 this
+]
+```  
+
+You can use any instance to submit jobs to the cluster. Jet will
+take care of distributing it across all instances.
+
+## Docker compose example
+
+When the configuration is more involved, including networks, volumes
+etc. it is usually easier to create a `docker-compose.yml` file. It also
+provides an advantage that you can easily scale the Jet cluster:
+
+Create a file called `docker-compose.yml`:
+
+```yaml
+version: '3'
+
+services:
+  jet:
+    image: hazelcast/hazelcast-jet
+    ports:
+      - "5701-5703:5701"
+```
+
+And then start 3 node cluster by running the following command:
```

Comment:
... 3-node cluster ...

Note: I am seeing the usage of "node" and "member" throughout the whole Jet documentation. Maybe it is better to come up with a convenience. IMDG uses "member".